xlators/features/trash/src/trash.c

```diff
@@ -2501,7 +2512,8 @@ init(xlator_t *this)
         goto out;
     }
 
-    priv->trash_itable = inode_table_new(0, this, 0, 0);
+    if (priv->state)
+        priv->trash_itable = inode_table_new(0, this, 0, 0);
```

Comment:
Shouldn't we return an error if the inode table has not been created ?

Reply:
done